docs/UserGuide.md

```diff
@@ -173,30 +173,31 @@ Examples:
 
 ### Finding persons by contact information : `find`
 
-Finds persons whose names, email address, contact number, or tag contain any of the given keywords.
+Finds persons whose name, email address, contact number, or tags contain any of the given keywords, even if the keyword is only a part of the information stored.
 
 Format: 
 
 `find PREFIX/KEYWORD [PREFIX/MORE_KEYWORDS]…​`
 
 * Only the specified fields are searched.
-* The search is case-insensitive. e.g. `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `n/Hans n/Bo` will return `Hans Gruber`, `Bo Yang`
+* The search is case-insensitive. e.g. `hans` will match `Hans`.
+* The order of the keywords does not matter, e.g. `find n/Hans n/Bo` is equivalent to `find n/Bo n/Hans`.
+* Persons matching at least one keyword will be returned (i.e. `OR` search),
+  e.g. `find n/Hans n/Bo` will match persons with names `Hans Gruber` or `Bo Yang`.
+* Keywords do not need to fully match the field, a partial match is sufficient, e.g. `bern` can match names like `Bernice` or `Bernard`
 
 How `find` works with multiple parameters:
 * If multiple keywords for a field are present, `find` will display contacts that satisfy *any* keyword for that field.
 * If multiple fields are present, `find` will display contacts that satisfy **all** of these fields.
 
 Examples:
 * `find n/bernice t/is1108` returns `Bernice Yu`<br>
-  ![result for 'find n/bernice t/is1108'](images/findBerniceIS1108Result.png){ width="750px" }
+  ![result for 'find n/bernice t/is1108'](images/findBerniceIS1108Result.png){ width="675px" }
 
 * `find t/floor t/run` returns all contacts with any tag containing `floor` or `run` <br>
-  ![result for 'find t/floor t/run'](images/findFloorRunResult.png){ width="750px" }
+  ![result for 'find t/floor t/run'](images/findFloorRunResult.png){ width="675px" }
 
-<div style="page-break-after: always;"></div>
+* `find n/alex n/bernice t/floor t/neighbour` returns all contacts that have names containing either `alex` or `bernice` **with** tags containing either `floor` or `neighbour`
```

Comment:
Please review this section especially

Reply:
I believe the description now is much clearer